third-party/snyk/snapshot.json

```diff
@@ -96,6 +97,7 @@
       {"id":"npm:knockout:20130701","severity":"medium","semver":{"vulnerable":[">=2.1.0-pre <3.0.0"]}}
     ],
     "lodash":[
+      {"id":"SNYK-JS-LODASH-567746","severity":"medium","semver":{"vulnerable":["*"]}},
```

Comment:
we also don't allow wildcards it should be `<=${latestVersion}`.

Reply:
we have a test that checks this (I'm sure you recall) but seems like really we'd want to also drop or fix this constraint automatically ... otherwise, unless we fix the source we're gonna be blocked indefinitely.

Reply:
Yeah their script *has* been doing this since we introduced that check, but I'm a little fuzzy on whether they're actually running our `cleanup-vuln-snapshot.js` or not and whose responsibility it is to update any potential bugs there 🤔

Reply:
@patrickhulce - we're definitely running the `cleanup-vuln-snapshot.js` script on our side before opening the PR, but I don't see any replacement of `*` there. I'm looking at https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/scripts/cleanup-vuln-snapshot.js (and this is the one we're running). Did I miss it?

Reply:
I wouldn't expect that script to be doing that, all it does is prune the non-detectable ones. 

I assumed something on the snyk side was replacing `*` with the latest npm version before opening because they stopped appearing for quite some time after we raised our concerns.

Reply:
I'll look into that.

Reply:
@patrickhulce from what I could find, we've never actively replaced `*`s with versions. Can you shortly explain your need here? 
Our concern is around new versions released without the vuln being fixed, and this will cause the scan to not pick those up.

Reply:
We discussed and agreed upon this policy last year, see https://github.com/GoogleChrome/lighthouse/issues/8409#issuecomment-485009237

> this will cause the scan to not pick those up.

That is the desired outcome. Lighthouse versions often get used for months to years after they're released. Heck Lighthouse doesn't even *reach* stable Chrome until ~2-3 months after release, so if a vulnerable library is patched quickly we would *always* be incorrectly flagging fixed versions to our users.

We only want statements in this snapshot that are always true, e.g. it is a fact that versions before `X.X.X` suffer from this vulnerability and always will.

Reply:
thank you @patrickhulce. We'll make sure we only send a `<=x.y.z` versions in the future. 
Sorry for missing this requirement.

Reply:
No worries! I think it's kind of awesome that it just accidentally happened to be working out for almost a full year 😆 

Thanks for being flexible working with us on this 👍

Reply:
Anytime! 🤗